MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -5,6 +5,8 @@
 import UIKit
 
 public final class FeedViewController: UITableViewController, UITableViewDataSourcePrefetching {
+	@IBOutlet private(set) var errorView: ErrorView!
```

Comment:
I believe the `errorView` is never accessed outside the class scope, so the setter can also be private:

```suggestion
	@IBOutlet private var errorView: ErrorView!
```